zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -886,206 +886,211 @@ property, when available, is noted below.
-       be sent with txn, when follower tries to read the digest, it will throw EOF which 
-       is caught and handled gracefully with digest value set to null.
-    3. When loading old snapshot with new code, it will throw IOException when trying to
-       read the non-exist digest value, and the exception will be caught and digest will
-       be set to null, which means we won't compare digest when loading this snapshot, 
-       which is expected to happen during rolling upgrade
-    4. When loading new snapshot with old code, it will finish successfully after deserialzing 
-       the data tree, the digest value at the end of snapshot file will be ignored
-    5. The scenarios of rolling restart with flags change are similar to the 1st and 2nd 
-       scenarios discussed above, if the leader enabled but follower not, digest value will
-       be ignored, and follower won't compare the digest during runtime; if leader disabled
-       but follower enabled, follower will get EOF exception which is handled gracefully.
-
-    Note: the current digest calculation excluded nodes under /zookeeper 
-    due to the potential inconsistency in the /zookeeper/quota stat node, 
-    we can include that after that issue is fixed.
-
-    By default, this feature is enabled, set "false" to disable it.
-
-* *snapshot.compression.method* :
-    (Java system property: **zookeeper.snapshot.compression.method**)
-    **New in 3.6.0:**
-    This property controls whether or not ZooKeeper should compress snapshots
-    before storing them on disk (see [ZOOKEEPER-3179](https://issues.apache.org/jira/browse/ZOOKEEPER-3179)).
-    Possible values are:
-    - "": Disabled (no snapshot compression). This is the default behavior.
-    - "gz": See [gzip compression](https://en.wikipedia.org/wiki/Gzip).
-    - "snappy": See [Snappy compression](https://en.wikipedia.org/wiki/Snappy_(compression)).
-
-* *snapshot.trust.empty* :
-    (Java system property: **zookeeper.snapshot.trust.empty**)
-    **New in 3.5.6:**
-    This property controls whether or not ZooKeeper should treat missing
-    snapshot files as a fatal state that can't be recovered from.
-    Set to true to allow ZooKeeper servers recover without snapshot
-    files. This should only be set during upgrading from old versions of
-    ZooKeeper (3.4.x, pre 3.5.3) where ZooKeeper might only have transaction
-    log files but without presence of snapshot files. If the value is set
-    during upgrade, we recommend to set the value back to false after upgrading
-    and restart ZooKeeper process so ZooKeeper can continue normal data
-    consistency check during recovery process.
-    Default value is false.
-
-* *audit.enable* :
-    (Java system property: **zookeeper.audit.enable**)
-    **New in 3.6.0:**
-    By default audit logs are disabled. Set to "true" to enable it. Default value is "false".
-    See the [ZooKeeper audit logs](zookeeperAuditLogs.html) for more information.
+ * *serverCnxnFactory* :
```

Comment:
Could we try to minimize the unintentional format change so it will be less confusing?